packages/api-fetch/README.md

```diff
@@ -22,6 +22,15 @@ apiFetch( { path: '/wp/v2/posts' } ).then( ( posts ) => {
 	console.log( posts );
 } );
 
+// GET with Query Args
+import { addQueryArgs } from '@wordpress/url';
```

Comment:
Can we move this import the top of the "file"? Below the `apiFetch` import.

Reply:
What if I break it into 2 code blocks? The second import is not necessary if not adding args.